code/script.js

```diff
@@ -1,19 +1,136 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+alert(`Welcome to Hungry Hugo's! Ready to Start? - Click 'OK' to begin.`);
+
+const userName = prompt("Please enter your name", "Hungry Hugo");
+console.log("user name", userName);
```

Comment:
Please remove the console.log() around the code. Generally, they need to be deleted when the code has been tested and works as meant to 👍